code/modules/clothing/chameleon.dm

```diff
@@ -437,6 +437,22 @@
 /obj/item/weapon/storage/backpack/chameleon/emp_act(severity)
 	chameleon_action.emp_randomise()
 
+/obj/item/weapon/storage/belt/chameleon
+	name = "toolbelt"
+	desc = "Holds tools."
```

Comment:
make sure that it looks like a real item in its default state, otherwise it's a metatell